.eslintrc.json

```diff
@@ -15,7 +15,7 @@
 	"settings": { "react": { "version": "detect" } },
 	"rules": {
 		"no-duplicate-imports": "warn",
-		"no-unused-vars": "warn",
+		"no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
 		"react/jsx-filename-extension": [1, { "allow": "as-needed" }],
```

Comment:
this was my solution as I had to loop over sets and didn't know how to avoid this here:
```
	const getClassName = (name) => {
		const statusArray = Object.entries(urgency).find(([_, items]) => {
			return Array.from(items).some((item) => item.name === name);
		});
		if (!statusArray) {
			throw new Error(`Failed to get class name of ${name}`);
		}
		return statusArray[0];
	};
```